pages/Product/components/Reviews/components/List/components/Review/index.jsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) 2017, Shopgate, Inc. All rights reserved.
+ *
+ * This source code is licensed under the Apache 2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './style';
+import Title from './components/Title';
+import Rating from './components/Rating';
+import Text from './components/Text';
+import Info from './components/Info';
+
+/**
+ * Single Review Component
+ * @param {Object} props The props
+ * @returns {JSX}
+ */
+const Review = ({ review }) => (
+  <div className={styles.review}>
+    <Title title={review.title} />
+    <Rating rate={review.rate} />
+    <Text review={review.review} />
+    <Info review={review} />
+  </div>
+);
+
+Review.propTypes = {
+  review: PropTypes.shape(),
+};
+
+Review.defaultProps = {
```

Comment:
Because you only show Reviews from the parent component if there is at least one, the `review` parameter will always have content here. Therefore a default prop is not needed. Please remove the whole definition block here.

Reply:
Done.